lms/djangoapps/grades/events.py

```diff
@@ -258,3 +260,21 @@ def fire_segment_event_on_course_grade_passed_first_time(user_id, course_locator
         )
 
     log.info("Segment event fired for passed learners. Event: [{}], Data: [{}]".format(event_name, event_properties))
+
+
+def openedx_persistent_grade_summary_changed(course_id, user_id, grade):
+     # .. event_implemented_name: PERSISTENT_GRADE_SUMMARY_CHANGED
+        PERSISTENT_GRADE_SUMMARY_CHANGED.send_event(
```

Comment:
the tabulation here seems one off.

Reply:
Yes, that's right, I forgot to indent it, We don't need this comment anymore, so I deleted it.